Switch EKS guide to use operator. (#4501)
projectcalico/calico: maintenance/ebpf/ebpf-and-eks.md

        - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
  EOF
  ```
* Install the Tigera operator:
  ```
  kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
  ```

  > **Note**: Do not use the manifests from the [<u>current EKS docs</u>](https://docs.aws.amazon.com/eks/latest/userguide/calico.html)

>   > because they reference an older version of Calico, which does not support eBPF mode.

Reviewer: This statement is likely to go out of date at some point.  Maybe:
```suggestion
  > because they reference an older version of Calico, which does not have the latest eBPF fixes.
```
